assets/BreakingNewsAsset.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace humhub\modules\breakingnews\assets;
+
+use yii\web\AssetBundle;
+
+class BreakingNewsAsset extends AssetBundle
+{
+    public $sourcePath = '@breakingnews/resources';
+    public $css = [];
+    public $js = [
+        'js/humhub.breakingnews.js'
+    ];
+
+    public $publishOptions = [
+        'forceCopy' => true,
```

Comment:
Can you please set this to "false". We automatically delete all assets during module updates.